web-ng/src/app/components/layer-list/layer-list.component.html

```diff
@@ -22,4 +22,10 @@
   <div *ngFor="let entry of layers">
     <ground-layer-list-item [layer]="entry"> </ground-layer-list-item>
   </div>
+  <div class="add-layer">
+    <button mat-raised-button class="add-layer-btn">
```

Comment:
Please use `<button mat-stroked-button color="primary">` instead to match mocks more closely.